src/main/java/stepsdesign/PipelineSteps.java

```diff
@@ -659,4 +665,18 @@ public void clickOnCreateButton () {
   public void verifyInvalidConnectionErrorMessage(String errorMessageLocation) {
     CdfConnectionActions.verifyConnectionErrorMessage(errorMessageLocation);
   }
+
+  @Then("Click on the Plus Green Button to import the pipelines")
+  public void clickOnPlusGreenButton () {
+    CdfConnectionActions.clickOnAddEntityButton();
+  }
+
+  @Then("Select the json files for importing the pipelines for the plugin {string}")
```

Comment:
Discuss this with Poojan once instead of writing the step for Json file you can make this step generic by creating the Utility class and adding a method for file upload.

Reply:
resolved